rca/project_styleguide/templates/patterns/pages/programmes/programme_index.html

```diff
@@ -1,22 +1,27 @@
 {% extends "patterns/base_page.html" %}
 {% load wagtailcore_tags wagtailimages_tags navigation_tags static %}
 
+{% block header_class %}
+    theme-light
+{% endblock %}
+
 {% block content %}
     <div class="page">
 
         <header class="page__header bg bg--dark">
-            <div class="title-area grid">
+            <div class="title-area title-area--index grid">
                 <div class="title-area__content">
-                    <h1 class="title-area__header heading heading--display-two">{{ page.title }}</h1>
-                    <p class="title-area__introduction introduction">{{ page.introduction|richtext }}</p>
+                    <h1 class="title-area__header heading heading--display-two">{{ title_multiline|linebreaksbr }}</h1>
```

Comment:
1. TIL `linebreaksbr`

2. Unsure of the state of the pattern library (or if this template is going to be included in it due to how the listings will work) but could add `title_multiline` to the `yaml` config file

Reply:
Good point! The template is in it now so I’ll add this.